### ISSUE TITLE
Recommendations from screen reader accessibility research 

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#27446]
## Issue Description
This epic will house the recommended following the screenreader usability study in July 2021. 
[Research findings](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/facilities/facility-locator/research/user-research/screenreader-usability-study/research-findings.md)

### Short Term

- [ ]  Assess and refine page hierarchies for Facility Locator  department-of-veterans-affairs/va.gov-team#27659 
    - Establish a page hierarchy (page outline with intended H1s, H2s, H3s, regions etc)
      - Eg. COVID content and links should move into an alert region that can be announced as such (what is the VA pattern to follow, work with accessibility/design) department-of-veterans-affairs/va.gov-team#27447
      - Are map, results listing, search areas properly named?
    - Test pages for consistency between different navigation methods (e.g. by heading level, skip to content, etc). Try [headingsMap](https://chrome.google.com/webstore/detail/headingsmap/flbjommegcjonpdmenkdiocclhjacmbi?hl=en) to verify.
- [ ]  Ensure ‘Skip to content’ link go to the page H1 ([see W3 for reference](https://www.w3.org/TR/WCAG20-TECHS/G1.html)) on Facility Locator, its detail pages as well as new facility product pages department-of-veterans-affairs/va.gov-team#27445
- [ ]  Service type combo box / Community Providers (in VA's network) improvements:
    - Update the hint text to accurately describe the action a user must need to take before the long term fix can be made department-of-veterans-affairs/va.gov-team#27449
    - What type aheads are available or need to be designed for VA.gov?
- [ ]  Restore results text when no results are found after a search department-of-veterans-affairs/va.gov-team#25629 
- [ ]  Assess why “Get directions” is not announcing properly on screen readers and propose solution department-of-veterans-affairs/va.gov-team#25759
     - Update screen reader text to inform users that link 1. goes to Google maps, 2. in a new tab
     - Document the pattern and assess other facility pages to follow
- [ ] Update ‘phone’ links' to ensure SRs users know that the number is a phone number (work with accessibility specialist) department-of-veterans-affairs/va.gov-team#27451
    - Assess/add TTY department-of-veterans-affairs/va.gov-cms#13939
    - Document the pattern and assess other facility pages to follow
- [ ] Make listing letter and mileage improvements department-of-veterans-affairs/va.gov-team#27452
    -  Work with accessibility specialists to assess the current problems with the letter and mileage combination and suggest improvements.
    - Check with other teams who are experimenting with other info hierarchy patterns (Dene/Peter)
      - Would numbered listings be more natural to hear when announced by a screen reader (and see by sighted users)?
      - Mileage could be better labeled and described by screen readers (e.g. "miles away" vs "miles")
        - Ensure improvement is for all users; sighted, cog disabilities
      - Work with other teams who are testing other patterns
      - Is the placement of these elements optimal?
- [ ]  Assess and incorporate keywords. Veterans don’t think to look for a facility locator and are likely to use global search when looking for a facility. department-of-veterans-affairs/va.gov-team#27702 Work with other teams to:
    - Assess keyword searches for top facility types on VA.gov and incorporate those terms into global search best bets
    - Experiment with patterns that exist within popular search tools like ‘hospitals near me’, ‘urgent care near me’ that could appear in type ahead global searches to connect users from the homepage to Facility Locator.
- [ ]  Does it have an ER with 24/7 care? This should be a data point that gets surfaced when available.

### Medium Term

Add ‘Google as the front door for Facility Locator and facility product pages’ as a use case to solve for and create an action plan. department-of-veterans-affairs/va.gov-team#2776
- [ ] Pre-work, deciding the type of facility, etc 
- [ ] Assess keyword searches for top facility types on VA.gov and incorporate those terms into the product.
    - E.g. Multiple users mentioned searching for burial and cemetery information and urgent care. Top terms could be surfaced in the page description.
- [ ]  Start experimenting with structured content for location addresses and hours to see how changes could positively affect search engine search results. 
    - Start with one small change to track and learn from
      - consider incorporating experiment into Vet Center pages where we have more research, control and editor involvement
- [ ]  Investigate how structured health data (or other subtypes) reconciles with current health taxonomy
    - Start with one small change to track and learn from
      - If a VA has ER care with 24/7 availability could be a data point be updated to conform to [google structured data](https://developers.google.com/search/docs/advanced/structured-data/intro-structured-data) and schema.org
 [ ]  Service type combo box / Community Providers (in VA's network) improvements: department-of-veterans-affairs/va.gov-team#27450
    - Identify clinician/SME expertise to inform this.
    - Restart mapping work (consider consolidating or grouping service type options). 

### Long term (Hypotheses to test)

- [ ]  Reassess what is the most important information to deliver on Facility locator. Based on the last year of research and with product guidance, might we try a hack-a-thon to explore alternative designs to test with Veterans? Things to question:
    - Is a map delivering value to Veterans? Veterans use other means to map directions later in their decision-making journey and it is unlikely that they would come to Facility Locator to do that. Could the mileage data point and ‘Get directions’ be enough without displaying a map or perhaps reducing its screen allocation 
      - (e.g. ⅓ vertical strip like Yelp or AirBnb)? If so, part or all of this real estate could be better allocated to information that would be more useful to Veterans when comparing between facilities.
    - Redesign search. Veterans want an easier way to search. In this study, participants didn’t want to mess with all the “requirements” and their expectations were heavily influenced by their search engine searches.
      - Consider a set of exposed filters in a left rail with default selections in place so that users only have to enter in their location, what would be more accessible and useful to all Veterans?
    - List hours over facility status alerts. An alert like ‘Limited hours and services’ does not provide enough information to help a Veteran choose a facility and instead compels a Veteran to call. Knowing a facility's operating hours would.
    - Give Veterans a way to quickly assess facilities at a glance. When Veterans search for VAs for long term planning like for travel or moving they are cognizant of the stark variations between size and scope of facilities across the country but they have no way of distinguishing them easily. For example, they want to know when a facility is a small clinic with limited services, if they can use it for their common primary care needs, or if there is 24/7 ER availability which is crucial in an emergency situation. 
   - Services; how might we display just enough service information to give Veterans a quick understanding of the type of facility to answer, “is this just a little clinic?”
    - Assess repeated links that don’t provide facility specific information. We learned how important it is to reduce the number of elements that a user has to hear. Is there a better way to provide access to general benefit information like for urgent care community care benefits?
    - Reassess the language used on Facility Locator. ‘Facility type’ and ‘Service type’ were not easily understood and it annoyed users to have to parse their meaning.
      - Consider creating meaningful tags, or categories to assign to facilities.
    - Consider creating a key with explanations for terms that may be confusing or link to another page for more information.
Improve discoverability of facility information available to them. Veterans are unaware of the different facility types that they can find using Facility Locator. Consider a more discoverable way of browsing.
      - For example, a set of exposed filters in a left rail could surface facility types, like cemeteries, that might be unknown but useful to more Veterans. Two Veterans in this study described recent searches for burial and cemetery information.

**Comment by jilladams** [comment]
Moved the final issue to https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12524. Closing.
